ts/features/itwallet/issuance/screens/ItwIssuanceCredentialFailureScreen.tsx

```diff
@@ -145,6 +146,10 @@ const ContentView = ({ failure }: ContentViewProps) => {
 
   useEffect(() => {
     if (storedCredential) {
```

Comment:
Can we return earlier if no credential is found? So we can avoid the nesting of `if` statements.